### PR TITLE
[PT-BR] Soundness.md

### PR DESCRIPTION
**Pull request by omurilo**
Hi there!

I had to make some adaptations in the translation, very small, that's how I managed to execute it.

I didn't translate line 80 because I couldn't understand the general context of it, I even asked @khaosdoctor for help, but she's really confused.

I really appreciate the opportunity and ask you to review the PR. @khaosdoctor, @alvarocamillont, @danilofuchs e @orta

### CODE REVIEW

File: packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
+//// {compiler: { strictFunctionTypes: false } }
+
+// Sem um histórico na teoria de tipos, é improvável que você
+// esteja familiarizado com a ideia de um sistema de tipos ser "sólido".
+
+// Solidez é a ideia de que o compilador pode dar garantias sobre o tipo
+// de um valor em tempo de execução, e não apenas durante a compilação.
+// Isso é normal para a maioria das linguagens de programação
+// que são construídas com tipos desde o primeiro dia.
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts]:
```suggestion
// que são construídas com tipos desde a sua criação.
```

File: packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+// Construir um sistema de tipo que modela uma linguagem que existe
+// há algumas décadas, no entanto, torna-se sobre tomar
+// decisões com trocas em três qualidades: Simplicidade,
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts]:
```suggestion
// decisões que podem ter efeitos em três propriedades: Simplicidade,
```

File: packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+// Isso pode voltar ao menor tipo comum:
+
+escuteOEvento("mouse", (event: {}) => {});
+
+// Mas não mais:
+
+escuteOEvento("mouse", (event: string) => {});
+
+// Isso cobre o padrão real do ouvinte de eventos em JavaScript,
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts]:
```suggestion
// Isso cobre o padrão real do event listener em JavaScript,
```

File: packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+// às custas de ser sólido.
+
+// O TypeScript pode gerar um erro quando isso acontecer por meio
+// de `strictFunctionTypes`. Ou você pode contornar este caso
+// específico com sobrecargas de função, consulte:
+// exemplo: typing-functions
+
+// Void special casing
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts]:
```suggestion
// Casing especial para Void
```

File: packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+// Descarte de parâmetros
+
+// Para aprender sobre casos especiais com parâmetros de função,
+// consulte o exemplo: structural-typing
+
+// Parâmetros restantes
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts]:
```suggestion
// Parâmetros rest
```

File: packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+// Os parâmetros restantes são considerados opcionais, isso significa
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts]:
```suggestion
// Os parâmetros rest são considerados opcionais, isso significa
```

File: packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts [hunk 7]
```diff
@@ -0,0 +1,116 @@
+//// {compiler: { strictFunctionTypes: false } }
+
+// Sem um histórico na teoria de tipos, é improvável que você
+// esteja familiarizado com a ideia de um sistema de tipos ser "sólido".
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts]:
```suggestion
// esteja familiarizado com a ideia de um sistema de tipos ser "sólido" (_sound_).
```

File: packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+
+// Solidez é a ideia de que o compilador pode dar garantias sobre o tipo
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Typescript/Language/Soundness.ts]:
```suggestion
// Solidez (_Soundness_) é a ideia de que o compilador pode dar garantias sobre o tipo
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
I tried to make some changes so it makes a bit more sense in some words that are more common in their original language.

**Comment by khaosdoctor** [comment]
@omurilo Are you still with us? Can you please fix these changes so we can merge the PR? 😄

**Comment by omurilo** [comment]
I hadn't seen your comments, I just saw this one today.

I'm here, I'm making the changes to update the PR.

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**